packages/dx-react-grid/docs/guides/export.md

```diff
@@ -0,0 +1,94 @@
+# React Grid - Export
+
+The Grid component supports exporting data in Excel format.
+
+## Related Plugins
+
+- [Toolbar](../reference/toolbar.md) - renders the Grid Toolbar
+- [ExportPanel](../reference/export-panel.md) - renders the Export Panel
+
+Note that [plugin order](./plugin-overview.md#plugin-order) is important.
+
+## Related Components
+
+- [GridExporter](../reference/grid-exporter.md) - performs data export
+
+## Basic Setup
+
+1. Import GridExporter
+
+Import the [GridExporter](../reference/grid-exporter.md) component and place it outside the Grid. Specify the `ref` property and add the `columns` and `rows` properties to the GridExporter.
+
+```jsx
+const exporterRef = useRef();
+...
+<Grid>
+...
+</Grid>
+<GridExporter
+  ref={exporterRef}
+  columns={columns}
+  rows={rows}
+/>
+```
+
+2. Import ExportPanel
+
+Import the [ExportPanel](../reference/export-panel.md) plugin and add it to your Grid.
+
+3. Initiate export
+
+Call the GridExporter's `exportGrid` method using a ref from the first step. This method can be called in the `ExportPanel`'s `startExport` callback or anywhere else in your code.
+
+```jsx
+exporterRef.current.exportGrid();
+```
+
+4. File saving
```

Comment:
Be careful these all numbers are `1` on the site.
![image](https://user-images.githubusercontent.com/15093892/73533781-aa082180-4430-11ea-8a64-3e37e6fb549e.png)